### ISSUE TITLE
Fix all "Double-quoted include in framework header, expected angle-bracketed instead" warnings

### ISSUE THREAD

**Issue by lmmenge** [onfido/onfido-ios-sdk#232]
### What was the expected behaviour?

There should be no warnings when using default Xcode's warning configuration.

### What happened instead?

Xcode 12.5 will suggest turning on by-default the following warning: CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER=YES.

```
Warns when a quoted include is used instead of a framework style include in a framework header.
```

Turning it on throws warnings for all the places the Onfido framework is using a project (quote) include instead of a framework (angle bracket) import.

### Version info:

- Cocoapods (if applicable): N/A
- Onfido SDK (Debug/Release): 21.4.0 Release
- iOS: N/A
- Xcode: 12.5
- Device/Simulator: N/A
- Device/Simulator language: N/A

### Integration configuration:

Manual Xcframework integration into a project using Xcode 12.5.

### Steps to reproduce:

Just use Xcode 12.5 on a clean project with all the recommended Xcode 12.5 warnings on or upgrade an old project to have all the Xcode 12.5 recommended warnings.

**Comment by andra-radu** [comment]
Hello @lmmenge Apologies for the late reply. We have been trying to reproduce the issue by following the steps but unfortunately we weren't abel to reproduce the issue.
1. create a new project using (Xcode 12.5, XCode13) [Minimuim version iOS 12]
2. add Onfido SDK to the project using (SwiftPackage manager, Cocoapods)
3. import the SDK and write the initial snippet for SDK configuration
4. build the project successfully with no warnings.

Are you still encountering this issue?

**Comment by lmmenge** [comment]
@andra-radu Yes I am. I've created a sample repository here https://github.com/lmmenge/onfido-double-quote which you can open and build in Xcode 13. The errors should be pretty in your face.

<img width="1392" alt="Screen Shot 2021-10-16 at 16 20 14" src="https://user-images.githubusercontent.com/51199/137591048-50395f21-9110-492c-b9d4-2e0d29b9041c.png">
